src/injected/ui.cpp

```diff
@@ -2799,33 +2799,63 @@ PresentPtr &vtable_find(T *obj, int index) {
     return *reinterpret_cast<PresentPtr *>(&ptr[0][index]);
 }
 
-void init_script() {
+void spawn_door_lua(float x, float y, uint8_t l, uint8_t w, uint8_t f, uint8_t t)
+{
+    spawn_door(x, y, l, w, 1, t + 1);

```

Comment:
Why even take `f` in the lua wrapper if you're just going to pass one? And why do you add 1 to t? Is there some reason the lua api should behave differently from the game api? If there's a good reason for it, that's something worth commenting because otherwise it's just confusing.

Reply:
I believe http://www.lua.org/manual/5.3/manual.html#6.1 covers the scope of what gets pulled in for base. I'm not sure we want to make all of that available just to provide `print` personally

Reply:
Talking in Discord it seems it's because the all themes are +1 from the enum. Seems to make more sense to me for the enum be updated to start at 1 instead of +1 everywhere.